Update docs for LAST_INSERT_ID() (#19831)
pingcap/docs: functions-and-operators/information-functions.md

>
> - In the preceding example, IDs increase by 2 while MySQL would generate IDs incrementing by 1 in the same scenario. For more compatibility information, see [Auto-increment ID](/mysql-compatibility.md#auto-increment-id).

> The `LAST_INSERT_ID(expr)` function can be used with an expression as argument, in which case the value is remembered for the next call to `LAST_INSERT_ID()` allowing this to be used as a MySQL compatible way of generating sequences. Note that TiDB also has support for proper [sequences](/functions-and-operators/sequence-functions.md).

Reviewer: ```suggestion
The `LAST_INSERT_ID(expr)` function can accept an expression as an argument, storing the value for the next call to `LAST_INSERT_ID()`. This allows it to be used as a MySQL-compatible method for generating sequences. Note that TiDB also supports proper [sequences](/functions-and-operators/sequence-functions.md). 
```
